src/main/java/beefy/command/DeleteCommand.java

```diff
@@ -0,0 +1,36 @@
+package beefy.command;
+
+import beefy.BeefyException;
+import beefy.storage.Storage;
+import beefy.task.TaskList;
+
+import java.io.IOException;
+
+public class DeleteCommand implements Command {
+    private TaskList userTasks;
+    private int taskId;
+
+    public DeleteCommand(TaskList userTasks, String userParams) throws BeefyException {
+        this.userTasks = userTasks;
+        try {
+            taskId = Integer.parseInt(userParams);
+        } catch (NumberFormatException e) {
+            throw new BeefyException("Use format:" + System.lineSeparator() + "delete (taskId)");
+        }
+    }
+
+    @Override
+    public void execute() throws BeefyException, IOException {
+        if (taskId < 1 || taskId > userTasks.getNumberOfTasks())
+        {
+            throw new BeefyException("Can you not do math, mate?");
+        }
```

Comment:
Try to use K&R style for if statements. Perhaps, move the brackets based on the coding standards.